features/facebookImport/src/components/onOffFacebookMiniStory/onOffFacebookMiniStory.jsx

```diff
@@ -26,19 +26,42 @@ const legend = (
 
 export const OnOffFacebookMiniStorySummary = ({
     companiesCount,
-    purchasesCount,
+    companiesWithAdsCount,
```

Comment:
I'm not clear about this change. It's not mentioned within the scope of the PR, it might be because it's not rebased with main?

Reply:
I guess I have to improve my commit messages? We (or better: the design)  just wanted to have a different value there, that's why (Andrei and) I changed it.